pkg/asset/rhcos/image.go

```diff
@@ -163,6 +164,17 @@ func osImage(config *types.InstallConfig) (string, error) {
 		return osimage, nil
 	case none.Name:
 		return "", nil
+	case nutanix.Name:
+		op := config.Platform.Nutanix
```

Comment:
What does `op` stand for?

Reply:
openshift platform. Simplified the conditional to avoid temp variables.